scripts/build/builders/bouffalolab.py

```diff
@@ -107,6 +107,12 @@ def __init__(self,
         elif enable_shell:
             self.argsOpt.append('chip_build_libshell=true')
 
+        try:
+            self.argsOpt.append('bouffalolab_sdk_root="%s"' % os.environ['BOUFFALOLAB_SDK_ROOT'])
+        except KeyError:
+            # environment BOUFFALOLAB_SDK_ROOT is existed, try default setup path
```

Comment:
Lets not do this: this path is specific to our docker images. There is no reason to assume it is generally a good default.

Let the build fail and the user set the required SDK.

Reply:
In third_party/bouffalolab/repo sub-module, there is a script required to execute to install Bouffalo Lab SDK on physical machine for none-docker development, default path is /opt/bouffalolab_sdk. 
Both readme in Matter-bouffalolab example and third_party/bouffalolab/repo mention to require to do this during environment setup.

Reply:
This comment was regarding having uniform handling of builders code. For that purpose, I insist that you do not just blindly set a default.

Reply:
Trying on default path is removed. 
For developer, `export BOUFFALOLAB_SDK_ROOT=/opt/bouffalolab_sdk` is required before building executed, which is mentioned in example Readme.md.